index.html

```diff
@@ -7631,6 +7631,12 @@ <h3><dfn data-lt="adding a cookie">Add Cookie</dfn></h3>
    <dd><p>The value if the entry exists, otherwise leave unset to
     indicate that this is a session cookie.
 
+    <aside class=note>
+      <p>The cookie’s expiry value may be limited by the remote end
+       according to an implementation-defined cap, as permitted by
+       the [[RFC6265bis]].</p>
```

Comment:
```suggestion
      <p>The cookie’s expiry value may be limited by the remote end
       as required by [[RFC6265bis]].</p>
```

But also, can we link to https://httpwg.org/http-extensions/draft-ietf-httpbis-rfc6265bis.html#name-cookie-lifetime-limits specifically rather than to the general RFC, I think that would make things clearer.

Reply:
Updated to a direct link. Once the draft is merged we probably want to add a new ref.